src/modules/services_manager_service.rb

```diff
@@ -46,10 +47,11 @@ def initialize
     # @param name [String] service name
     # @return [Yast2::SystemService, nil]
     def find(name)
-      return services[name] unless Stage.initial
+      return services[name] unless Stage.initial || Mode.auto
 
-      # We are in inst-sys. So we cannot check for installed services but generate entries
-      # for these services if they do not exist yet.
+      # In inst-sys we cannot check for installed services but generate entries for these
+      # services if they do not exist yet. The same applies to AutoYaST when a service might be
+      # installed as part of the software selection.
       services[name] = Yast2::SystemService.build(name)
```

Comment:
Is this always creating a new `SystemService` with each call to `#find` ? I mean, in first stage and AutoYaST mode.

Reply:
Yes, that's it. Usually we work with short services lists (we are importing a profile) and, as far as I know, it is not a performance problem.